packages/cubejs-athena-driver/src/AthenaDriver.ts

```diff
@@ -126,16 +126,20 @@ export class AthenaDriver extends BaseDriver implements DriverInterface {
       getEnv('athenaAwsSecret', { dataSource });
 
     const { schema, ...restConfig } = config;
+    
+    let { credentials } = config;
+    
+    if (typeof credentials === undefined) {
+      credentials = accessKeyId && secretAccessKey ? { accessKeyId, secretAccessKey } : undefined;
+    }
 
     this.schema = schema ||
       getEnv('dbName', { dataSource }) ||
       getEnv('dbSchema', { dataSource });
 
     this.config = {
       ...restConfig,
-      credentials: accessKeyId && secretAccessKey
-        ? { accessKeyId, secretAccessKey }
-        : undefined,
+      credentials,
```

Comment:
I suggest simplifying this code by moving the original code for `credentials`:

```
credentials: accessKeyId && secretAccessKey
        ? { accessKeyId, secretAccessKey }
        : undefined,
```

before `...restConfig`.

It will allow anyone to redeclare `credentials` by passing it via `constructor`

Reply:
<img width="451" alt="image" src="https://github.com/cube-js/cube/assets/572096/3cdae808-f8a7-46d7-ab58-5c38a208e26f">